src/components/ContentView/ContentView.tsx

```diff
@@ -18,11 +20,48 @@ export type ContentViewProps = {
  */
 export const ContentView: React.FC<ContentViewProps> = ({ contentData, pageTitle }) => {
     const [content, setContent] = useState<ContentData>({ ...contentData } as ContentData);
-
+    const [activeSection, setActiveSection] = useState({section: "", index: 0, numSections: 0});
+    const sections: string[] = [];
```

Comment:
I feel like there may be some sort of state issue with using this without the `useState` hook. Can you confirm if this is fine?